src/Components/AppbarBottom/AppbarBottom.svg.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Dimensions } from 'react-native';
+import Svg, { G, Path, Rect } from 'react-native-svg';
+
+const AppbarBottomSVG = ({
+  paddingHorizontal,
+  width,
+  backgroundColor,
+  fabPosition,
+}) => {
+  if (!width) {
+    width = Dimensions.get('screen').width - paddingHorizontal;
+  }
+
+  let rectWidth = width / 2 - 92 / 2;
```

Comment:
Break off whatever 92 is into a constant so if it needs updating you only have to do it in one spot.